### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by akaeyuhi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+  7 - monobank exchange rates
+  8 - Recent Crypto News
+  9 - PrivatBank exchange rates
+  10 - Cryptocurrency fee rates
+  11 - Show currency code by its number and vice versa
+  12 - Transaction info by its hash
+  13 - Exchange rates of UAH from bank.gov.ua (Alternative)
+  Type anything to exit.`);
+  const selection = parseInt(await question('Select action\n')) - 1;
+  let features = [
```
Comment (tshemsedinov) [inline index.js]: Use const here

File: index.js [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+    exchanges.btcAdrBalance,
+    exchanges.monoExchange,
+    crypto.cryptoNews,
+    exchanges.privatExchange,
+    exchanges.feesRate,
+    exchanges.currencyCodeNumber,
+    crypto.transactionInfo,
+    exchanges.nbuAlternative,
+  ];
+  features = features.map(item => item.bind(crypto));
```
Comment (tshemsedinov) [inline index.js]: Don't rewrite variables, just reflect with another variable

File: src/crypto.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const promised = require('./promised.js');
+const green = '\x1b[32m';
+const red = '\x1b[31m';
```
Comment (tshemsedinov) [inline src/crypto.js]: Colors in `crypto` module, really?
Ubp37r9

File: src/crypto.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      console.log(`${resultText.join('\n')}\n`);
+      await safeWrite(resultText);
+    }
+    return resultText.join('\n');
+  }
+
+  async topFiveCurrencies() {
+    const query = this.defaultUrl + '/top/totalvolfull?limit=10&tsym=USD';
+    const currencies = (await safeGet(query)).Data;
+    currencies.splice(4, 5);
```
Comment (tshemsedinov) [inline src/crypto.js:39]: Magic numbers
giphy-downsized-large

File: src/crypto.js [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+    }
+    return resultText.join('\n');
+  }
+
+  async cryptoNews() {
+    const info = await safeGet(`${this.defaultUrl}/v2/news/?lang=EN`);
+    const data = info.Data;
+
+    let proposedTitles = '\nFive most recent articles on cryptocurrency:\n';
+    for (let i = 0; i < 5; i++) {
```
Comment (tshemsedinov) [inline src/crypto.js]: Why 5? Magic

**Comment by akaeyuhi** [comment]
Done.

**Pull request closed without merging**